### PR TITLE
:rocket: New `/kubecon-demo` page

### LINKED ISSUE #3521: Add scheduling page to website
For more context see [Notion](https://www.notion.so/gitpod/Add-scheduling-page-to-website-9c19acddc67a42dd84264b3217cf92e8)

calendly embed link: https://calendly.com/d/yyt-j28-84m

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

The page would be live on `/kubecon-demo`. And, I will update the same page with a description, calendly URL, etc for every event. And, this would make URLs consistent and easily accessible at every time in the future.

## Related Issue(s)

Fixes #3521

## How to test

- Open [`/kubecon-demo`](https://deploy-preview-3524--gitpod-kumquat.netlify.app/kubecon-demo) & see :eyes:

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3524"></a>

### CODE REVIEW

File: src/routes/kubecon-demo.svelte [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  import UsedBy from "$lib/components/index/used-by.svelte";
+  import OpenGraph from "$lib/components/open-graph.svelte";
+</script>
+
+<OpenGraph
+  data={{
+    title: "Get a demo of Gitpod at KubeCon EU 2023",
+    description:
+      "Supercharge your productivity by how you develop. Get a demo of Gitpod's continuous development platform to see how to maximize your developer experience.",
+    keywords: "continuous development, remote development, demo, get started",
```
Comment (Siddhant-K-code) [inline src/routes/kubecon-demo.svelte:18]: @MichaelAring I've updated the `title`. But, still need your 👀 on it & also for `description` & `keywords`

File: src/lib/components/kubecon-demo/header.svelte [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <div>
+    <HeaderDescription
+      class="!m-0"
+      title="Let's meet at KubeCon EU"
+      label="Conference"
+      date="April 19-21"
+      location="Amsterdam, The Netherlands"
+      description="Our CEO and CTO will be at KubeCon. Schedule a meeting now to discuss running Gitpod at your company.<br/><br/>
+    We'll be at Booth SU22, RAI Convention Center. Come visit us in person or see us in our virtual booth online."
+    />
```
Comment (Siddhant-K-code) [inline src/lib/components/kubecon-demo/header.svelte:45]: @MichaelAring Please verify this description Text. Anything we wanted to add/remove?

**Review by chrifro — COMMENTED** [review]
- Could you adjust the viewport or size for the calendly embed so that the CTA is visible without scrolling? Otherwise, the UX could be very confusing. If not we could consider removing one question from the embed.
Bildschirm­foto 2023-03-20 um 08 27 12
- could you add a break before "KubeCon" in the title so that "kubeCon EU" is in one line? 

- "Emails of other attendees who will join *" should be optional -> will update that in calendly

**Comment by Siddhant-K-code** [comment]
> If not we could consider removing one question from the embed.

If it's possible, can we do that? Because even if we would change the height. It would be a longer form and difficult UX.

File: src/lib/components/kubecon-demo/header.svelte (hunk 2 above)
Comment (chrifro) [reply src/lib/components/kubecon-demo/header.svelte:45]: Let's change the last sentence to "We'll be at Booth SU22, come and visit us."

**Review by chrifro — APPROVED** [review]
I aligned with Michael and shortened the calendly embed. Let's make the suggested messaging change and merge it ✨

**Pull request merged**